development/idp-local/config/config.php

```diff
@@ -22,7 +22,7 @@
 
 try {
     // Required to be defined in environment variables
-    $ADMIN_EMAIL = Env::requireEnv('ADMIN_EMAIL');
+    $ADMIN_EMAIL = Env::get('ADMIN_EMAIL', 'na@example.org');
```

Comment:
As @mtompset noticed, since this one isn't required now, it doesn't match the comment explaining this section of the code. Could you move this `ADMIN_EMAIL` line down to somewhere around line 47, to be with the other optional env. vars?

Similarly in the other places an `Env::requireEnv()` was changed to an `Env::get()` with a fallback/default value.